test/Tests/src/Semantic/Python_Interop_Spec.enso

```diff
@@ -87,7 +90,18 @@ spec =
 
         Test.specify "should expose array interfaces for Python arrays" <|
             vec = Vector.from_polyglot_array make_array
-            vec.map .x . should_equal [10, 20, 30]
+            vec.map .x . should_equal [30, 10, 20]
+
+            arr = vec.map .x . to_array
+            sorted = arr.sort
```

Comment:
This is not a Python array anymore, right? After a `Vector.map` operation, this will be an Enso allocated Java array, no?

Still, a good test but this part is not really testing what it says - only the part below is _actually_ invoking our sort on a Python array instance.

Reply:
Indeed, yes.